include/zephyr/drivers/espi/espi.h

```diff
@@ -101,9 +101,9 @@ enum espi_io_mode {
  */
```

Comment:
Please don't delete the old file, we should instead leave an empty stub here with a warning that has:

```
#warning "<zephyr/espi.h> is deprecated, please include <zephyr/espi/espi.h>"
#include <zephyr/espi/emul.h>
```